group/secp256k1/scalar_test.go

```diff
@@ -0,0 +1,127 @@
+package secp256k1
```

Comment:
I was hoping to see test cases that compare operations in this library to those in other libraries. Can you find out if there are a standard set of inputs and outputs that other secp256k1 implementations use to check compatibility between each other? I'm not saying that we have to have this, just that if such a thing is normally used, then we should be using it as well. Or you could be the first: generate a few scalars and points using libsecp256k1 and then confirm that the same results come from this library.

Reply:
I don't think there's anything standard, though of course there are test vectors we could borrow from other suites: [Python's cryptography package](https://github.com/pyca/cryptography/blob/c574e7554c7aa27c56f6478258a4e18f79457652/vectors/cryptography_vectors/asymmetric/ECDSA/SECP256K1/SigGen.txt), [some random vectors](https://chuckbatson.wordpress.com/2014/11/26/secp256k1-test-vectors/).